### ISSUE TITLE
Using Kaltura API with WordPress plugin

### ISSUE THREAD

**Issue by nicmitchell** [kaltura/all-in-one-video-pack.wordpress#6]
I have been unsuccessful in trying to use the Kaltura CE6 API to extend the functionality of the Kaltura WordPress plugin v2.5.

**TL;DR Questions:**
- How do I get the Kaltura API to work with the WordPress v2.5 plugin? 
- Is there a file I need to include/require? 
- Do I need to change how the class names are written?

**The Problem:**
I am trying to accomplish various things like pulling stats from the KMC and publish liveStream entries at various times.

Considering that WP loads plugins on every page, I am assuming that the Kaltura plugin tries to handle creating a Kaltura session for the user associated with the WP site.

However, I am not able to use any of the standard API calls generated in the KMC API Console. I always receive a fatal error of something like "Class 'KalturaConfiguration' not found in /path/to/file.php on line 999".

It appears that the class names are slightly different in the plugin than they are in the php client API. I found the TypeMap.php file and library, which tells me that there is an effort to translate the differences accordingly, but I have not found how to get that to load or function properly. I have tried to include/require various files such as:
/lib/Kaltura/Client/TypeMap.php
/lib/Kaltura/Client/Client.php
/lib/Kaltura/Client/ClientBase.php
/lib/Kaltura/Client/ServiceBase.php
/lib/Kaltura/Client/Configuration.php

... and various others. Still the same fatal error of no Class Name found.

I also installed the standard Kaltura php client library and included that. That causes a Fatal Error of "Cannot redeclare class KalturaNull". I would rather use the native API library of the plugin anyway.

How can I get them to work together?
